feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+this file was also added in thefeature1 branch
```

Comment:
it's really good that you have decided to add this line of code

Reply:
thanks for this